tests/conftest.py

```
@@ -143,3 +144,22 @@ def _make(
         )
 
     return _make
+
+
+@pytest.fixture
+def ohne_jitter():
+    """Setzt die Last-Glättung auf 0, damit Sollzeiten exakt prüfbar sind.
+
+    Die Jitter-Phase leitet sich aus der Subentry-ID ab, die je Testlauf neu
+    vergeben wird – ohne diesen Eingriff wäre nur ein Intervall prüfbar.
```

Ruff (0.16.2)

[warning] Docstring contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF002)